tests/splines/batched_spline_builder.cpp

```diff
@@ -142,13 +181,23 @@ struct DimsInitializer<IDimI, ddc::detail::TypeSeq<IDimX...>>
         ddc::init_discrete_space<IDimI>(
                 GrevillePoints<
                         BSplines<typename IDimI::continuous_dimension_type>>::get_sampling());
+#if defined(BC_HERMITE)
+        ddc::init_discrete_space(
+                IDimDeriv<ddc::Deriv<typename IDimI::continuous_dimension_type>>::
+                        init(Coord<ddc::Deriv<typename IDimI::continuous_dimension_type>>(1),
+                             Coord<ddc::Deriv<typename IDimI::continuous_dimension_type>>(
+                                     std::max(2, (int)s_degree_x / 2)),
```

Comment:
This is necessary because of the following static_assert:
https://github.com/CExA-project/ddc/blob/0326f51fccc7be0a8eec1ffca1082c49d6ae5eca/include/ddc/uniform_point_sampling.hpp#L117

Maybe we could relax that to a<=b ? This is a practical use-case where a=b get sense.

Reply:
My answer for that depends on my other comment.

Reply:
I don't know which comment

Reply:
About the value of `IDimDeriv`, if it needs initialization or not